resources/views/members/project/partials/form.blade.php

```diff
@@ -0,0 +1,32 @@
+<div class="row">
+  <label for="projectName" class="form-label">Name</label>
+  <div class="form-control">
+    <input id="projectName" type="text" name="projectName" value="{{ old('projectName', $project->getProjectName()) }}" required autofocus>
+    @if ($errors->has('projectName'))
+    <p class="help-text">
+      <strong>{{ $errors->first('projectName') }}</strong>
+    </p>
+    @endif
+  </div>
+</div>
+
+<div class="row">
+  <label for="description" class="form-label">Description</label>
+  <div class="form-control">
+    <textarea id="description" name="description" rows="10" required>{{ old('description', $project->getDescription()) }}
+    </textarea>
```

Comment:
This whitespace is included in the field.

should be <textarea ...>{{ blade-stuff }}</textarea>

all on one line.  Otherwise four spaces are added to the field each time the project is saved!